src/SFA.DAS.AssessorService.Application/Interfaces/IStandardRepository.cs

```diff
@@ -14,7 +14,6 @@ public interface IStandardRepository
         Task<StandardCollation> GetStandardCollationByReferenceNumber(string referenceNumber);
         Task<DateTime?> GetDateOfLastStandardCollation();
         Task<int> GetEpaoStandardsCount(string endPointAssessorOrganisationId);
```

Comment:
GetEpaoStandardsCount is used elsewhere. Left it in the Interface so that we can unit test it